pkg/api/v1/clients/configmap/tests/plain/plain_resource_client_test.go

```diff
@@ -14,20 +15,16 @@ import (
 
 	v1 "github.com/solo-io/solo-kit/test/mocks/v1"
 
-	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
-	"github.com/solo-io/go-utils/log"
 	. "github.com/solo-io/solo-kit/pkg/api/v1/clients/configmap"
 	"github.com/solo-io/solo-kit/test/helpers"
 	"k8s.io/client-go/kubernetes"
-
-	// Needed to run tests in GKE
-	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 )
 
 var _ = Describe("PlainConfigmap", func() {
 	if os.Getenv("RUN_KUBE_TESTS") != "1" {
-		log.Printf("This test creates kubernetes resources and is disabled by default. To enable, set RUN_KUBE_TESTS=1 in your env.")
+		fmt.Print("This test creates kubernetes resources and is disabled by default. To enable, set RUN_KUBE_TESTS=1 in your env.")
```

Comment:
per discussion above, I wasn't sure if the goal was to revert this back to `log.`, or leave as-is

Reply:
Ah good call I had missed these, wil update